### ISSUE TITLE
Issue with phpunit 4.7.* and hhvm

### ISSUE THREAD

**Issue by roman-shuhov** [sebastianbergmann/phpunit#1752]
I got an error 

```
Fatal error: Uncaught exception 'PharException' with message 'phar has a broken signature' in :
Stack trace:
#0 (): Phar->parsePhar()
#1 (): Phar->__construct()
#2 /usr/local/bin/phpunit(15): Phar::mapPhar()
#3 {main}
```

when i tried to run phpunit 4.7.* phar with hhvm 3.7.*

the same phar file works fine with php 5.5

phpunit <= 2.6.9 doesn't have this issue.

**Comment by sebastianbergmann** [comment]
If it works with PHP and not HHVM then it is an HHVM issue.

**Comment by HRMsimon** [comment]
This has been fixed by facebook/hhvm#5858 and should be in HHVM 3.9.
